graspy/simulations/sims_sample_corr.py

```diff
@@ -0,0 +1,91 @@
+import numpy as np
+from graspy.simulations import sample_edges
+import copy
+import warnings
+
+def sample_corr(P, Rho, directed=False, loops=False):
+    """
+    Generate a correlated Erdos-Renyi graph G2 based on G1 graph
```

Comment:
I know the authors call this ER, but lets not specify that here, I think it's confusing